service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/no.symlink.to.directory.out.of.sandbox.rpt

```diff
@@ -0,0 +1,6 @@
+connect http://localhost:8701/../../../../linkToDirectoryNotAllowed/
```

Comment:
Not sure how this is different then continuous.symlinks.into.directory.from.outside.rpt